### ISSUE TITLE
Calendar ignores value of NavigationLowerBound

### ISSUE THREAD

**Issue by pjdavid** [ME-MarvinE/XCalendarGettingStarted#2]
Hi

I'm using the XCalendar in my Xamarin.Forms project.

I'm trying to set the values of 'NavigationLowerBound' and 'NavigationUpperBound' as part of initialising the calendar but the calendar is ignoring these values until an event forces the calendar to update, date selection or navigation. Once any of these events occurs the calendar behaves as expected and invalidate dates outside the 'NavigationLowerBound' and 'NavigationUpperBound'  range.

Thanks.

**Comment by ME-MarvinE** [comment]
Are you able to provide your initialisation code and XAML?

Is the Calendar being initialised in ViewModel or code-behind?

What version of XCalendar are you using?

**Comment by pjdavid** [comment]
The calendar is being initialised in my ViewModel and I'm using version 4.5.2, here is the code:
```c#
public Calendar<CalendarDay> MyCalendar { get; set; } = new Calendar<CalendarDay>()
        {
            SelectionType = SelectionType.None,
            SelectionAction = SelectionAction.Replace,
            NavigationLowerBound = DateTime.Today,
        };
```

        Thanks.

**Comment by ME-MarvinE** [comment]
@pjdavid 

Moved to the main [XCalendar](https://github.com/ME-MarvinE/XCalendar) repo: ME-MarvinE/XCalendar#171

**Comment by pjdavid** [comment]
Hi mate.

Sorry for the very late reply. Moved away from the project for a while.

I'm not sure the issue issue is fixed and the new version 4.6.0 it seems to made it a lot worse, or maybe I'm doing something wrong. Now all DayView are displaying today's and if I try to go to a previous date it takes me to 1999 (Please see attached).

Regards,
Paulo
![Screenshot_20240403-091208](https://github.com/ME-MarvinE/XCalendarGettingStarted/assets/1098447/0681c693-8c51-46e9-879e-3fe7bf6a73aa)

![Screenshot_20240403-093736](https://github.com/ME-MarvinE/XCalendarGettingStarted/assets/1098447/cd3a768b-8e28-49b8-a383-30b76bb25b24)

**Comment by ME-MarvinE** [comment]
Oops, forgot to lock the conversation. As mentioned in my previous comment,  I've moved this issue to the main XCalendar repo.

ME-MarvinE/XCalendar#171
